### PR TITLE
Remove decryption when retrieving secret from MLP

### PR DESCRIPTION
**Pull request by pradithya**
MLP's Secret API removes the application-level encryption when [returning the secret data.](https://github.com/caraml-dev/mlp/pull/85). Moving forward, we'll rely on TLS to secure the secrets while in transit. 
This PR updates the MLP client used by Turing and removes the decryption process when retrieving a secret from MLP. 

- [X] Tested in dev by deploying router which requires secret

### CODE REVIEW

File: infra/charts/turing/templates/_helpers.tpl [hunk 1]
```diff
@@ -186,7 +186,7 @@ MLPConfig:
   MLPEncryptionKey: {{ include "turing.mlp.encryption.key" . | quote }}
 {{ if .Values.tags.mlp }}
   MerlinURL: {{ printf "http://%s:8080/v1" (include "turing.merlin.name" .) }}
-  MLPURL: {{ printf "http://%s:8080/v1" (include "turing.mlp.name" .) }}
+  MLPURL: {{ printf "http://%s:8080" (include "turing.mlp.name" .) }}
```
Comment (krithika369) [inline infra/charts/turing/templates/_helpers.tpl:189]: Some docs / sample values need to be updated with the same change:
* https://github.com/caraml-dev/turing/blob/main/README.md?plain=1#L233
* https://github.com/caraml-dev/turing/blob/main/api/.env.development#L30
* https://github.com/caraml-dev/turing/blob/main/api/config-dev.yaml#L91

**Review by krithika369 — APPROVED** [review]
LGTM. Thanks, @pradithya !

**Review by krithika369 — APPROVED** [review]
LGTM. Thanks, @pradithya !

File: infra/charts/turing/templates/_helpers.tpl (hunk 1 above)
Comment (pradithya) [reply infra/charts/turing/templates/_helpers.tpl:189]: Updated

**Pull request merged**